modules/react/common/lib/theming/createCanvasTheme.ts

```diff
@@ -103,7 +101,7 @@ function calculateCanvasTheme(partialTheme: PartialCanvasTheme): CanvasTheme {
     direction: direction === ContentDirection.RTL ? direction : ContentDirection.LTR,
   };
 
-  return merge({}, defaultCanvasTheme, mergeable, extraFields) as CanvasTheme;
+  return getObjectProxy({...mergeable, ...extraFields}, defaultCanvasTheme);
```

Comment:
I think this is the right order for getObjectProxy...we want the mergeable fields and extrafields and use defaultCanvasTheme

Reply:
If you look at `getCanvasTheme.ts` we're doing that there!